### ISSUE TITLE
FactorGroup(G,N) not the image of the NaturalHomomorphism? 

### ISSUE THREAD

**Issue by cdwensley** [gap-system/gap#2392]
This issue is related to the discussion which preceded issue #2378, as I'm still having problems with isomorphic, but not identical, pc-groups. 
I guess I'm querying the accuracy of 39.18.2 in the reference manual, which states that FactorGroup(G,N) returns the image of the NaturalHomomorphismByNormalSubgroup(G ,N ). 
The following simple example (nothing special about this choice of G,N) indicates otherwise: 
gap> g := Group( ( 1, 2, 3, 4, 5, 6, 7, 8, 9,10,11,12),  ( 2,12)( 3,11)( 4,10)( 5, 9)( 6, 8) );;
gap> n := Group( ( 1,10, 7, 4)( 2,11, 8, 5)( 3,12, 9, 6),  ( 1, 7)( 2, 8)( 3, 9)( 4,10)( 5,11)( 6,12) );; 
gap> ok := IsNormal( g, n );
true
gap> nat := NaturalHomomorphismByNormalSubgroup( g, n );
[ (1,2,3,4,5,6,7,8,9,10,11,12), (2,12)(3,11)(4,10)(5,9)(6,8) ] -> [ f2, f1 ]
gap> f := FactorGroup( g, n ); 
Group([ f1, f2 ])
gap> f = Image( nat );
false
gap> f = Range( nat );
false
gap> f = ImagesSource( nat ); 
false
gap> mgi := MappingGeneratorsImages( nat );; 
gap> q := Group( mgi[2] ); 
<pc group with 2 generators>
gap> q = Image( nat );
true

**Comment by ThomasBreuer** [comment]
Yes, the manual entry for `FactorGroup` is wrong or at least misleading.

The two functions `NaturalHomomorphismByNormalSubgroup` and `FactorGroup`
take the group and its normal subgroup as arguments,
and obviously the factor group objects created by them could be identical
only at the cost of some caching and lookup,
which is in my opinion not generally desirable.

I think the manual should state that
whenever one wants to work with some factor group `F` of a group `G`
such that one wants to use also the homomorphism from `G` to `F`,
the recommended approach is to create the homomorphism first,
and to access the factor group as the image of this homomorphism.

Thus `FactorGroup` is useful only in situations where one is sure
that one never wants to carry back some information from `F` to `G`.

Of course this advice holds also for other kinds of factor structures and for images of actions.
